### ISSUE TITLE
Website request - Make headers bolder/darker

### ISSUE THREAD

**Issue by jhibbets** [ncopenpass/NCOpenPass#1296]
I was using a combination of H2's and H3's in this post and the bold stands out more than the headers. Can we increase the font-weight on Header tags?

http://ncopenpass.com/?p=3999&preview=true

**Comment by jtawes** [comment]
Absolutely, if you are using elementor it's much more customizable. Do you
want me to go ahead and edit the draft and then you can see how I do it?

**Comment by jhibbets** [comment]
I would prefer not to use Elementor for blog posts. I think it's a barrier
to get people writing content unless there I'm just not missing out on the
advantage of it. I'm used to very simple WYSIWYG with simple, elegant
styles for writing. I don't think we need anything fancy for blog content.

**Comment by jtawes** [comment]
http://ncopenpass.com/?p=3999&preview=true

**Comment by jtawes** [comment]
Sorry about that, I did have an updated version to preview as you requested but when I switched it back to the WordPress editor, all of the styling was deleted.

I can only customize the style of the content with either custom CSS coding or with Elementor, otherwise you have the default styles from the theme. You don't have to create all blog posts with Elementor but if you want me to customize the style of a particular post, such as this one, then I need to switch to that. 

It doesn't prevent authors from creating new posts with the WordPress editor, it just allows for customization such as this when you want it.

**Comment by jtawes** [comment]
You can navigate to the post and click on the Edit with Elementor button in the middle of the screen to see a preview. If no changes, it's ready to publish.

**Comment by jtawes** [comment]
I can also work on customizing the default theme styles by editing the code but this is much faster for me at the moment and I'm short on time.

**Comment by jhibbets** [comment]
I think customizing the theme for headers and URLs would be great. I played
around with this early on as the URLs were hard to read. I made them green,
but didn't put much thought into and to be honest, was flying by the seat
of my pants trying to figure out the theme CSS.
